src/core.jl

```diff
@@ -86,18 +110,42 @@ struct Path <: Drawable
 end
 
 """
-    ellipse = Ellipse(center, ρx, ρy)
+    ellipse = Ellipse(center, ρx, ρy; thickness = 0,fill = false)
 
 A `Drawable` ellipse with center `center` and parameters `ρx` and `ρy`
+with keyword arguments `thickness` and `fill` for drawing hollow ellipse
 
+# Arguments
+- `thickness::Integer`: thickness of the ellipse
```

Comment:
```suggestion
- `thickness::Int`: thickness of the circle
```

Reply:
Only test that needs to be resolved is the UInt8 one,I think we can work on those keywords for other shapes after Polygon filling work is done....that would give us more perspective and how to best implement those keywords :)

Reply:
WARNING: Method definition (::Type{ImageDraw.CirclePointRadius{T} where T<:Real})(ImageDraw.Point, T<:Real) where {T<:Real} in module ImageDraw at /home/ashwani/Desktop/ImageDraw.jl/src/core.jl:87 overwritten at /home/ashwani/Desktop/ImageDraw.jl/src/core.jl:88.
  ** incremental compilation may be fatally broken for this module **
  Also this issue needs to be fixed before merging

Reply:
Ah, okay. This means that we also need test cases for `CartesianIndex`. I only see `Point` version used in the tests.

Reply:
@johnnychen94 Can you point out which cases I need to check for cartesian index?
```julia
        res = @inferred draw(img, CirclePointRadius(Point(5,5), 5), Gray{N0f8}(0.5))
        expected = map(i->(i==1 ? Gray{N0f8}(0.5) : Gray{N0f8}(0)), expected)
        @test all(expected .== res) == true
```
I am assuming this one